<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pagedlod_editor</class>
 <widget class="QWidget" name="pagedlod_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PagedLOD editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/TreeView/LOD.png</normaloff>:/icons/TreeView/LOD.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ranges</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Range 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_lineedit_rangemin1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="_lineedit_rangemax1"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Range 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_lineedit_rangemin2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="_lineedit_rangemax2"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Bounds</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Radius</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_lineedit_radius"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Center</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="_combobox_centermode">
          <item>
           <property name="text">
            <string>USE_BOUNDING_SPHERE_CENTER</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>USER_DEFINED_CENTER</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_lineedit_center_x">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_lineedit_center_y">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_lineedit_center_z">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
